README.md

```diff
@@ -46,6 +54,17 @@ After installation, launch PEtabGUI from the command line using the
 Optionally, you can provide the path to an existing PEtab YAML file
 as an argument.
 
+### Examples
+
+A set of real-world PEtab problems is available in the
+[PEtab benchmark collection](https://github.com/Benchmarking-Initiative/Benchmark-Models-PEtab).
+
+Download and unpack the [archive](https://github.com/Benchmarking-Initiative/Benchmark-Models-PEtab/archive/refs/heads/master.zip),
+then open one of the YAML files inside `Benchmark-Models/*/` using
+PEtabGUI.
+`Benchmark-Models/Boehm_JProteomeRes2014/Boehm_JProteomeRes2014.yaml`
+should serve as good starting point.
```

Comment:
This is already our example, as it is not immediately downloaded via pip, I guess this is fine?


```suggestion
should serve as a good starting point.
```